src/operator/rnn-inl.h

```diff
@@ -50,10 +50,53 @@ namespace rnn_enum {
   enum RNNOpResource {kTempSpace};
 }
 
+inline int GetRnnParamSize(int num_layer,
+                           int input_size,
+                           int state_size,
+                           int direction,
+                           int mode) {
+  int size = state_size * direction;
+  switch (mode) {
+    case rnn_enum::kRnnRelu:
+    case rnn_enum::kRnnTanh:
+      size *= 1;
```

Comment:
why bother multiplying by 1?

Reply:
It's same as the old version, if you think it is confused, i'll remove it.